src/Users.php

```diff
@@ -259,10 +280,10 @@ private function getAuthToken($name = '', $salt = '')
             $seed .= '-' . $this->remoteIP;
         }
         if ($this->app['config']->get('general/cookies_use_browseragent')) {
-            $seed .= '-' . $_SERVER['HTTP_USER_AGENT'];
+            $seed .= '-' . $this->userAgent;
         }
         if ($this->app['config']->get('general/cookies_use_httphost')) {
-            $seed .= '-' . (isset($_SERVER['HTTP_HOST']) ? $_SERVER['HTTP_HOST'] : $_SERVER['SERVER_NAME']);
+            $seed .= '-' . $this->hostName;
```

Comment:
No, no, no need for dropping anything.. Just wanted to check that it wasn't an oversight. ^_^